conf_files/peas.yaml

```diff
@@ -14,7 +14,8 @@ environment:
 weather:
     station: mongo
     aag_cloud:
-        serial_port: '/dev/ttyUSB1'
+      #        serial_port: '/dev/ttyUSB1'
+        serial_port: '/dev/tty.USA19H2P1.1'
```

Comment:
We should probably just comment this out and force the use of `peas_local.yaml`. I've been thinking of doing that system wide to make it obvious to new installers what needs to be changed in the config. Should be okay for here, just mostly thinking out loud.